code/modules/jobs/job_types/security_officer.dm

```diff
@@ -26,6 +26,13 @@
 
 	family_heirlooms = list(/obj/item/book/manual/wiki/security_space_law, /obj/item/clothing/head/beret/sec)
 
+	mail_goodies = list(
+		/obj/item/food/donut/caramel = 10,
+		/obj/item/food/donut/matcha = 10,
+		/obj/item/food/donut/blumpkin = 5,
```

Comment:
throw in an oztek boomerang or a stingbang or something here

ooh, maybe one of those rotation grenades?